lib/dotiw/action_view_ext/helpers/date_helper.rb

```diff
@@ -1,7 +1,7 @@
 module ActionView
   module Helpers
     module DateHelper
-      alias_method :old_distance_of_time_in_words, :distance_of_time_in_words
+      alias old_distance_of_time_in_words distance_of_time_in_words
```

Comment:
Could you please provide more details behind this change?

Reply:
I ran `rubocop -a`, https://github.com/rubocop-hq/ruby-style-guide#alias-method-lexically

Reply:
I reverted this one in https://github.com/radar/distance_of_time_in_words/pull/101